### ISSUE TITLE
vapor (Toolbox) hanging / not ending

### ISSUE THREAD

**Issue by MichaelKofler** [vapor/vapor#1163]
* Vapor Version: 2
* OS: macOS
* OS Version: 10.12.6
* Vapor Toolbox Version: 2.0.4
* XCode: both 8.3.3 and 9.0 are installed, command line tools set to 8.3.3

### Expected Behavior

When running the `vapor` command in a terminal window, it should end or give an error.

### Current Behavior

`vapor` (the toolbox command) in many cases does not end. This affects commands such as `vapor new`, `vapor build` or `vapor xcode`. I guess it is in most cases actually done, but it continues running. No CPU load. I have to stop it with ctrl-C. In a few cases, it helps to add `--verbose`, but not always.

After a previously hanging vapor commando, not even `vapor --version` would end! It shows the version number and runs on, forever. In this case, I can quit the Terminal app. If I try again, `vapor --version` works as expected, but other vapor commands still hang.

The solution for this is rebooting macOS. Then `vapor` runs fine for a while until the problems comes back.

Basically I guess some component continues running, waiting for whatever. I don't know what process to look for. `ps -ax` shows nothing suspicious.

**Comment by tanner0101** [comment]
SPM 8.x has been known to hang indefinitely. Try w/ command line tools set to 9.0 and reopen if the error continues.
